osquery/killswitch/plugins/killswitch_json.h

```diff
@@ -0,0 +1,14 @@
+#include "osquery/remote/serializers/json.h"
+#include <osquery/killswitch/killswitch_refreshable_plugin.h>
+#include <string>
+
+namespace osquery {
+
+class KillswitchJSON : public KillswitchRefreshablePlugin {
+ public:
+ protected:
+  virtual Status refresh() override;
```

Comment:
When you define something as `override` it is already `virtual`, so you don't need to put `virtual` here. It is redundant.